addons/account/migrations/12.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,387 @@
+NEW ir.ui.view: account.dashboard_onboarding_company_step
+NEW ir.ui.view: account.digest_digest_view_form
+NEW ir.ui.view: account.email_compose_onboarding_sample_invoice
+NEW ir.ui.view: account.invoice_tree_with_onboarding
+NEW ir.ui.view: account.onboarding_bank_account_step
+NEW ir.ui.view: account.onboarding_chart_of_account_step
+NEW ir.ui.view: account.onboarding_fiscal_year_step
+NEW ir.ui.view: account.onboarding_invoice_layout_step
+NEW ir.ui.view: account.onboarding_sale_tax_step
+NEW ir.ui.view: account.onboarding_sample_invoice_step
+NEW ir.ui.view: account.portal_invoice_chatter
+NEW ir.ui.view: account.report_payment_receipt_document
+NEW ir.ui.view: account.res_company_form_view_inherit_account_intrastat
+NEW ir.ui.view: account.setup_bank_account_wizard
+NEW ir.ui.view: account.view_account_position_filter
+NEW ir.ui.view: account.view_company_partner_bank_form
+NEW ir.ui.view: account.view_incoterms_tree
+DEL ir.ui.view: account.account_aged_balance_view
+DEL ir.ui.view: account.account_journal_dashboard_kanban_view_js
+DEL ir.ui.view: account.account_move_line_reconcile_writeoff
+DEL ir.ui.view: account.account_planner
+DEL ir.ui.view: account.account_report_balance_view
+DEL ir.ui.view: account.account_report_general_ledger_view
+DEL ir.ui.view: account.account_report_partner_ledger_view
+DEL ir.ui.view: account.account_report_print_journal_view
+DEL ir.ui.view: account.accounting_report_view
+DEL ir.ui.view: account.accounting_tax_report_view
+DEL ir.ui.view: account.portal_invoice_report
+DEL ir.ui.view: account.report_agedpartnerbalance
+DEL ir.ui.view: account.report_financial
+DEL ir.ui.view: account.report_generalledger
+DEL ir.ui.view: account.report_overdue
+DEL ir.ui.view: account.report_overdue_document
+DEL ir.ui.view: account.report_partnerledger
+DEL ir.ui.view: account.report_tax
+DEL ir.ui.view: account.report_trialbalance
+DEL ir.ui.view: account.setup_bank_journal_form
+DEL ir.ui.view: account.setup_opening_move_lines_tree
+DEL ir.ui.view: account.setup_opening_move_wizard_form
+DEL ir.ui.view: account.setup_posted_move_form
+DEL ir.ui.view: account.setup_view_company_form
+DEL ir.ui.view: account.view_account_financial_report_form
+DEL ir.ui.view: account.view_account_financial_report_search
+DEL ir.ui.view: account.view_account_financial_report_tree
+DEL ir.ui.view: account.view_account_invoice_filter_inherit_credit_notes
+DEL ir.ui.view: account.view_account_invoice_filter_inherit_invoices
+DEL ir.ui.view: account.view_account_move_line_reconcile_full
+DEL ir.ui.view: account.view_wizard_multi_chart
+DEL ir.ui.view: account_cash_basis_base_account.view_tax_form_cash_basis_inherit
+DEL ir.ui.view: account_invoicing.assets_backend
+DEL ir.ui.view: account_invoicing.product_template_form_view_invoice_policy
+DEL ir.ui.view: account_invoicing.res_config_settings_view_form_inherit_account_invoicing
+DEL mail.template: account.mail_template_data_notification_email_account_invoice (noupdate)
```

Comment:
We should remove this record by hand with `delete_record_safely...` from openupgradelib